### DISCUSSION TITLE
Is it possible to import ddlc save data if it already has been merged

### DISCUSSION THREAD

**Discussion by Plunon** [Monika-After-Story/MonikaModDev#8181] [QA - tech support + general]
I didn't have my save file when I installed the mod, so mod automatically thought i got true ending which is deleting monika and stuff It wasnt a problem back in early stages of the mod so I never thought about that but now options like "are you still mad at me for deleting you" or dialouge like "I know you won't leave me again" shows up so It gets me upset since I never did I dont remember importing save data but it says you already did  can I still load my save without losing progress or maybe by editing persistent I can make it like I didn't delete monika don't know if thats possible though

**Comment by Booplicate** [comment]
You can merge saves only once, you probably did it before at some point.
